lib/matplotlib/mpl-data/fonts/ttf/local.conf

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
-<!-- /etc/fonts.conf file to configure system font access -->
-<fontconfig>
-        <!--  Enable sub-pixel rendering
-        <match target="font">
-                <test qual="all" name="rgba">
-                        <const>unknown</const>
-                </test>
-                <edit name="rgba" mode="assign"><const>rgb</const></edit>
-        </match>
-         -->
-
-        <alias>
-                <family>serif</family>
-                <prefer>
-                        <family>Bitstream Vera Serif</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>sans-serif</family>
-                <prefer>
-                        <family>Bitstream Vera Sans</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>monospace</family>
-                <prefer>
-                        <family>Bitstream Vera Sans Mono</family>
-                </prefer>
-        </alias>
-</fontconfig>
```

Comment:
Regarding shipping them both, I just realised that even though the DejaVu fonts are 5.3 MB, the Vera fonts are only 600 KB, no from a distribution size perspective it is not a huge difference shipping them both rather than removing Vera.